test/Groups/Permutations.jl

```diff
@@ -427,3 +427,41 @@ end
   H,iso = smaller_degree_permutation_representation(G)
   @test degree(H)<degree(G)
 end
+
+@testset "fixed_points tests" begin
+  # Setup permutation group
+  g = symmetric_group(4)             # S₄
```

Comment:
No unicode (the comment is redundant anyway)
```suggestion
  g = symmetric_group(4)
```